apps/vaporgui/PSliderEdit.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "PLineItem.h"
+#include "PDynamicMixin.h"
+//#include "VaporWidgetsFwd.h"
+
+class VSliderEdit;
+
+
+//! \class PSliderEdit
+//! Creates a Qt slider and text input combo synced with the paramsdatabase using the PWidget interface.
+//! \copydoc PWidget
+
+class PSliderEdit : public PLineItem, public PDynamicMixin {
```

Comment:
Are there other PWidgets that make use of the PDynamicMixin?

Reply:
No, currently only PSliderEdit uses it.

Reply:
Thanks.  I'm fine resolving this.  Do you think the mixin should be used by some or all other PWidgets?

Reply:
Only those that would benefit from supporting dynamic updates.